frontend/src/views/editUser/EditUser.jsx

```diff
@@ -0,0 +1,402 @@
+                name="jobTitle"
+                onChange={handleChange}
+                value={formData.jobTitle}
+              />
+              {dummy.errors.jobTitle && (
+                <FormHelperText error>{dummy.errors.jobTitle}</FormHelperText>
+              )}
+            </div>
+            {dummy.gov ? (
+              <>
+                <div>
+                  <Typography fontSize={16}>IDIR User Name</Typography>
+                  <TextField
+                    fullWidth
+                    required
+                    error={!!dummy.errors.IDIRUserName}
+                    name="IDIRUserName"
+                    onChange={handleChange}
+                    value={formData.IDIRUserName}
+                  />
+                  {dummy.errors.IDIRUserName && (
+                    <FormHelperText error>
+                      {dummy.errors.IDIRUserName}
+                    </FormHelperText>
+                  )}
+                </div>
+                <div>
+                  <Typography fontSize={16}>Email address</Typography>
+                  <TextField
+                    fullWidth
+                    required
+                    error={!!dummy.errors.email}
+                    name="email"
+                    onChange={handleChange}
+                    value={formData.email}
+                  />
+                  {dummy.errors.email && (
+                    <FormHelperText error>{dummy.errors.email}</FormHelperText>
+                  )}
+                </div>
+              </>
+            ) : (
+              <>
+                <div>
+                  <Typography fontSize={16}>BCeID Userid</Typography>
+                  <TextField
+                    fullWidth
+                    required
+                    error={!!dummy.errors.BCeIDUserID}
+                    name="BCeIDUserID"
+                    onChange={handleChange}
+                    value={formData.BCeIDUserID}
+                  />
+                  {dummy.errors.BCeIDUserID && (
+                    <FormHelperText error>
+                      {dummy.errors.BCeIDUserID}
+                    </FormHelperText>
+                  )}
+                </div>
+
+                <div>
+                  <Typography fontSize={16}>
+                    Email address associated with the BCeID user account
+                  </Typography>
+                  <TextField
+                    fullWidth
+                    required
+                    error={!!dummy.errors.email}
+                    name="email"
+                    onChange={handleChange}
+                    value={formData.email}
+                  />
+                  {dummy.errors.email && (
+                    <FormHelperText error>{dummy.errors.email}</FormHelperText>
+                  )}
+                </div>
+                <div>
+                  <Typography fontSize={16}>
+                    Alternate email for notifications (optional)
```

Comment:
In the wireframe, the field titles are bold, but "(Optional)" is not. For example:

> **Alternate email for notifications** (optional)